pkg/project/registry/projectrequest/delegated/sample_template.go

```diff
@@ -36,11 +39,25 @@ func DefaultTemplate() *templateapi.Template {
 
 	binding := &authorizationapi.RoleBinding{}
 	binding.Name = "admins"
-	binding.Namespace = "${" + ProjectNameParam + "}"
+	binding.Namespace = ns
 	binding.Users = util.NewStringSet("${" + ProjectAdminUserParam + "}")
 	binding.RoleRef.Name = bootstrappolicy.AdminRoleName
 	ret.Objects = append(ret.Objects, binding)
 
+	serviceAccountsBinding := &authorizationapi.RoleBinding{}
+	serviceAccountsBinding.Name = "service-accounts"
```

Comment:
The binding should indicate what it's binding to, not what group you've bound.  The `RoleRef` is fixed, the users and groups are not.   I'm willing to fix in followups.